README.md

```diff
@@ -55,92 +50,128 @@ body {
 }
 ```
 
-will give you:
+when applaying the following Import Map:
+
+```json
+{
+    "imports": {
+        "normalize.css": "https://cdn.eik.dev/normalize.css@8/normalize.css",
+    },
+}
+```
+
+one will get a transformed CSS like so:
 
 ```css
-@import 'https://unpkg.com/normalize.css@8/normalize.css';
+@import 'https://cdn.eik.dev/normalize.css@8/normalize.css';
 
 body {
     background: black;
 }
 ```
 
-## PostCSS Import Usage
+## Options
 
-If you're using [postcss-import](https://github.com/postcss/postcss-import) make sure you update the `plugins` option.
-`postcss.config.js`
+This plugin takes the following as options:
+
+| option  | default        | type     | required | details                                                     |
+| ------- | -------------- | -------- | -------- | ----------------------------------------------------------- |
+| path    | `cwd/eik.json` | `string` | `false`  | Path to eik.json file.                                      |
```

Comment:
I think the `path` option should be a folder with either the eik.json or eik in a package.json. Perhaps `configDir` is a better name?

Reply:
I'm trying to align these values and behaviour across the plugins. and we're using `path` for the same in the other plugins so I think we'll stick with that. Updated the doc.